watcher/Dockerfile

```diff
@@ -1,25 +1,37 @@
-FROM lsstts/develop-env:b32
+FROM lsstts/deploy-env:salobj_v5.4.0_idl_v1.1.2_xml_v4.7.0
 
-LABEL maintainer Tiago Ribeiro <tribeiro@lsst.org>
+RUN git clone https://github.com/lsst-ts/ts_config_ocs.git
 
-USER saluser
+ENV TS_CONFIG_OCS_DIR=/home/saluser/ts_config_ocs
 
-WORKDIR /home/saluser/repos
-RUN git clone https://github.com/lsst-ts/ts_watcher.git
+WORKDIR /home/saluser/
 
-WORKDIR /home/saluser/repos/ts_watcher
-RUN git fetch --all && git checkout "v1.0.1" -b "v1.0.1"
+RUN source /home/saluser/.setup.sh && \
+    conda install -c lsstts ts-watcher="1.0.2rc1"
 
-WORKDIR /home/saluser/repos/ts_watcher
-RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
-    source /home/saluser/repos/ts_sal/setup.env && \
-    eups declare -r . ts_watcher -t current && \
-    setup ts_salobj -t current && \
-    setup ts_watcher -t current && \
-    scons
 
-WORKDIR /home/saluser/
+#FROM lsstts/develop-env:b32
```

Comment:
Why are we keeping this around?

Reply:
I am not brave enough to get rid of it yet. 😨